odoo/addons/base/tests/test_configmanager.py

```diff
@@ -565,6 +570,132 @@ def test_06_cli(self):
             'limit_request': 100,
         })
 
+    def test_06_environ(self):
+        with file_open('base/tests/config/environ') as file:
+            os.environ.update({
+                x[0]: x[2].rstrip('\n')
+                for line in file.readlines()
+                if line and (x := line.partition('=')) and x[0]
+                and not x[0].startswith('#')
+            })
+        self.config._load_env_options()
+        self.config._parse_config()
```

Comment:
This one should be enough
```suggestion
        self.config._parse_config()
```